src/CitMovie.Models/DomainObjects/Media.cs

```diff
@@ -1,5 +1,44 @@
 namespace CitMovie.Models.DomainObjects;
+[Table("media")]
 public class Media
 {
+    [Key, Column("media_id")]
+    public int MediaId { get; set; }
+    [Column("type")]
+    public string Type { get; set; }
+    [Column("plot")]
+    public string? Plot { get; set; }
+    [Column("runtime")]
+    public string? Runtime { get; set; }
+    [Column("box_office")]
+    public string? BoxOffice { get; set; }
+    [Column("budget")]
+    public string? Budget { get; set; }
+    [Column("imdb_id")]
+    public string? ImdbId { get; set; }
+    [Column("website")]
```

Comment:
I think we removed the website did we not?

Reply:
Hmm true, I was checking based on the ER we had in suproject 1 :D

Reply:
changed

Reply:
Have you pushed, can't still see it?